creating_packages/package_approaches.rst

```diff
@@ -93,54 +91,44 @@ library, in the same package.
+    A complete working example of the following code can be found in the examples repo: https://github.com/conan-io/examples
 
     .. code:: bash
 
-        $ git clone https://github.com/memsharded/hello_multi_config
-        $ cd hello_multi_config
-        $ conan create . user/channel -s build_type=Release
-        $ conan create . user/channel -s build_type=Debug --build=missing
+        $ git clone https://github.com/conan-io/examples.git
+        $ cd features/multi_config
+        $ conan create . user/channel
 
-Creating a multi-configuration Debug/Release package is simple, see the following example
-using CMake.
+Creating a multi-configuration debug/release package is simple
 
-The first step will be to remove ``build_type`` from the settings. It will not be an input setting, the
-generated package will always be the same, containing both Debug and Release artifacts.
-The Visual Studio runtime is different for debug and release (``MDd`` or ``MD``) and is set using the default runtime (MD/MDd). If this meets your needs, we recommend removing the ``runtime`` subsetting in the
-``configure()`` method:
+The first step will be to remove ``build_type`` from the settings. It will not be an input setting and the generated package will always
+contain both debug and release artifacts.
 
+The Visual Studio runtime is different for debug and release (``MDd`` or ``MD``) and is set using the default runtime (``MD``/``MDd``). If
+this meets your needs, we recommend removing the ``compiler.runtime`` subsetting in the ``configure()`` method:
 
 .. code-block:: python
 
-    class Pkg(ConanFile):
+    class HelloConan(ConanFile):
         # build_type has been omitted. It is not an input setting.
         settings = "os", "compiler", "arch"
+        generators = "cmake"
 
+        # Remove runtime and use always default (MD/MDd)
         def configure(self):
-            # it is also necessary to remove the VS runtime
             if self.settings.compiler == "Visual Studio":
                 del self.settings.compiler.runtime
 
         def build(self):
-            cmake = CMake(self)
-            if cmake.is_multi_configuration:
-                cmmd = 'cmake "%s" %s' % (self.source_folder, cmake.command_line)
-                self.run(cmmd)
-                self.run("cmake --build . --config Debug")
-                self.run("cmake --build . --config Release")
-            else:
-                for config in ("Debug", "Release"):
-                    self.output.info("Building %s" % config)
-                    self.run('cmake "%s" %s -DCMAKE_BUILD_TYPE=%s'
-                            % (self.source_folder, cmake.command_line, config))
-                    self.run("cmake --build .")
-                    shutil.rmtree("CMakeFiles")
-                    os.remove("CMakeCache.txt")
-
-In this case, we assume that the binaries will be differentiated with a suffix in the Cmake
-syntax:
+            cmake_release = CMake(self, build_type="Debug")
```

Comment:
doesn't it require different build folders to be used? otherwise, I guess Release will just override Debug (or fail?)

Reply:
name of the generated artifacts are different, check the CMakeLists of the example